modules/ocf_kubernetes/templates/master/loadbalancer/haproxy.cfg.erb

```diff
@@ -0,0 +1,60 @@
+global
+  daemon
+  log localhost local0
+  # TODO: send to syslog
```

Comment:
Do you mean the remote syslog server here (i.e. `democracy`)? If so, I think just locally logging is enough, it should get sent on to the remote syslog server by the local rsyslog instance.